main.tf

```diff
@@ -0,0 +1,14 @@
+provider "aws" {
+  region = "us-east-2"
+  access_key = "access_key"
```

Comment:
this code will not work, how you pass secret here ?

Reply:
I have used access_key and secret_key while Running it on my machine.
but removed from mian.tf when pushed to Github.